src/uct/ib/rc/accel/rc_mlx5_ep.c

```diff
@@ -776,32 +777,37 @@ static void uct_rc_mlx5_ep_reset_qp(uct_rc_mlx5_ep_t *ep)
      */
     memset(&qp_attr, 0, sizeof(qp_attr));
     qp_attr.qp_state = IBV_QPS_RESET;
-    ret = ibv_cmd_modify_qp(txqp->qp, &qp_attr, IBV_QP_STATE, &cmd, sizeof(cmd));
+    ret = ibv_cmd_modify_qp(qp, &qp_attr, IBV_QP_STATE, &cmd, sizeof(cmd));
     if (ret) {
-        ucs_warn("modify qp 0x%x to RESET failed: %m", txqp->qp->qp_num);
+        ucs_warn("modify qp 0x%x to RESET failed: %m", qp->qp_num);
     }
 #else
-    (void)uct_rc_modify_qp(txqp, IBV_QPS_ERR);
+    (void)uct_ib_modify_qp(qp, IBV_QPS_ERR);
```

Comment:
IBV_QPS_RESET?

Reply:
it should be ERR, since RESET triggers mlx5_cq_clean() which we want to avoid